app/views/publishers/vacancies/edit_vacancy_sections/_edit_pay_package.html.slim

```diff
@@ -13,9 +13,10 @@
         - row.key text: t("jobs.annual_salary")
         - row.value text: vacancy.salary
 
-      - summary_list.row do |row|
-        - row.key text: t("jobs.actual_salary")
-        - row.value text: vacancy.actual_salary.presence || t("jobs.not_defined")
+      - unless vacancy.__getobj__.working_patterns == ["full_time"]
```

Comment:
I'm not too sure what `__getobj__` is doing there, but `vacancy.working_patterns` will do I think!

Reply:
The __getobj__ allows us to access working_patterns directly from the Vacancy model rather than the VacancyPresenter which maps them to a different format (which could potentially change and cause us issues in the future?)

Reply:
This line means we're find to call `working_patterns` on the presenter, I think.

```ruby
delegate :working_patterns, to: :model, prefix: true
```

Reply:
Oh, I just saw that the presenter has a working_patterns method as well as that delegator. Confused now!

Reply:
Why does it have two presenter methods for working_patterns AND a delegator :/

Reply:
Cesi tells me we can use the prefix specified in the delegator i.e. `model_working_patterns` to access the delegator which looks nicer than using `__getobj__`